litert/vendors/qualcomm/core/builders/depthwise_conv2d_op_builder.cc

```diff
@@ -57,6 +59,18 @@ std::vector<OpWrapper> BuildDepthwiseConv2dOp(
     reshape_op.AddOutputTensor(*reshaped_filter_tensor);
   }
 
+  TensorWrapper* conv2d_output_tensor = nullptr;
+
+  // If Conv2d fused activation is available, create a new tensor for the
+  // output of Conv2d, which is also the input of the activation op.
+  if (fused_activation != kLiteRtFusedActivationNone) {
+    conv2d_output_tensor = &tensor_pool.CreateNativeTensor(
+        outputs[kOutputIndex].get().GetDataType(), {},
```

Comment:
Also need to keep quantization param.
```suggestion
        outputs[kOutputIndex].get().GetDataType(), outputs[kOutputIndex].get().GetQuantParams(),
```

Reply:
`tensor_pool.CloneNativeTensorFrom` is more convenient in this case.